project/sites/imtac/themes/imtac_theme/templates/layout/page.html.twig

```diff
@@ -113,57 +105,73 @@
-            <nav aria-label="Skip links">
-              <a href="#post-content" class="skip-link visually-hidden focusable">
-                {{- 'Skip to results'|t -}}
-              </a>
-            </nav>
-            {{ page.search_facets }}
-          </header>
+    {% if search_page %}
+    <div class="content-sidebar-wrap search-page">
+      {% else %}
+      <main class="content-sidebar-wrap">
         {% endif %}
-        {% if page.content %}
-          <main id="post-content" {{ page.search_facets ? 'class="main-content"' : '' }}>
-            {{ page.content }}
-          </main>
+        {{ page.breadcrumb }}
+        {% if search_page %}
+          {{ drupal_block('page_title_block', wrapper=false) }}
         {% endif %}
-      {% if is_front %}
-        {% if page.bottom_widget_1 or page.bottom_widget_2 or page.bottom_widget_3 %}
-          <div id="bottom_wrapper" class="footer_block bottom_widget">
-            <div id="bottom-area" class="footer-area clearfix">
-              {% if page.bottom_widget_1 %}
-                <div class="column">
-                  {{ page.bottom_widget_1 }}
-                </div>
-              {% endif %}
-              {% if page.bottom_widget_2 %}
-                <div class="column">
-                  {{ page.bottom_widget_2 }}
-                </div>
-              {% endif %}
-              {% if page.bottom_widget_3 %}
-                <div class="column">
-                  {{ page.bottom_widget_3 }}
-                </div>
-              {% endif %}
+        {% if is_front %}
+          {% if drupal_config('system.site', 'frontpage_title') %}
+            <h1
+              class="{{ drupal_config('system.site', 'frontpage_show_title') ? 'container-wrap' : 'visually-hidden' }}">
+              {{ drupal_config('system.site', 'frontpage_title') }}
+            </h1>
+          {% endif %}
+        {% endif %}
+        <div id="content" {{ page.search_facets ? '' : 'class="main-content"' }}>
+          {% if page.search_facets %}
+            <header id="search-facets" class="sidebar-first-layout">
+              <nav aria-label="Skip links">
+                <a href="#post-content" class="skip-link visually-hidden focusable">
+                  {{- 'Skip to results'|t -}}
+                </a>
+              </nav>
+              {{ page.search_facets }}
+            </header>
+          {% endif %}
+          {% if page.content %}
+            <div id="post-content" {{ page.search_facets ? 'class="main-content" role="main"' : '' }}>
+              {{ page.content }}
             </div>
-          </div>
+          {% endif %}
+          {% if is_front %}
+            {% if page.bottom_widget_1 or page.bottom_widget_2 or page.bottom_widget_3 %}
+              <div id="bottom_wrapper" class="footer_block bottom_widget">
+                <div id="bottom-area" class="footer-area clearfix">
+                  {% if page.bottom_widget_1 %}
```

Comment:
Would it be worth adding a comment here to indicate what's supposed to be rendered here? page.bottom_widget_1 is likely autogenerated but doesn't tell me what is intended to seen here.

Reply:
They are autogenerated, we have them in the unity_theme template. I'm not entirely sure what the exact reasoning for them is but I have added a comment to them all on my understanding.